src/dcos_e2e/backends/_docker/__init__.py

```diff
@@ -96,6 +96,124 @@ def _get_fallback_storage_driver() -> DockerStorageDriver:
         return DockerStorageDriver.AUFS
 
 
+class DockerLoopbackVolume():
```

Comment:
I'm a little torn because, ignoring the CLI I don't think that this belongs as part of DC/OS E2E's library - it doesn't really fit in my concept of what E2E is for.

However, I do see value in having this in the CLI, as you have explained to me that this would be useful as part of the CLI.

Ideally for me this would be a separate library in itself, but that is probably not going to happen.

Therefore, let's discuss where this goes - is `backends.DockerLoopbackVolume` natural? I think that this implies that it is a backend which it is not.